src/plugins/console/public/application/containers/editor/monaco/monaco_editor_actions_provider.ts

```diff
@@ -71,6 +91,13 @@ export class MonacoEditorActionsProvider {
     editor.onDidContentSizeChange(async (event) => {
       await debouncedHighlightRequests();
     });
+
+    editor.onKeyUp((event) => {
```

Comment:
It seems that `onKeyUp` is making some jest tests fail. Could we use `editor.addAction` or `editor.addCommand` instead?